core/src/main/scala/cats/data/NonEmptyList.scala

```diff
@@ -219,6 +224,40 @@ final case class NonEmptyList[+A](head: A, tail: List[A]) {
   }
 
   /**
+    * Sorts this `NonEmptyList` according to an `Order` on transformed `B` from `A`
+    *
+    * {{{
+    * scala> import cats.data.NonEmptyList
+    * scala> import cats.instances.int._
+    * scala> val nel = NonEmptyList.of(('a', 4), ('z', 1), ('e', 22))
+    * scala> nel.sortBy(_._2)
+    * res0: cats.data.NonEmptyList[(Char, Int)] = NonEmptyList((z,1), (a,4), (e,22))
+    * }}}
+    */
+  def sortBy[B: Order](f: A => B): NonEmptyList[A] =
+    toList.sortBy(f)(Order[B].toOrdering) match {
```

Comment:
I think we generally use an implicit parameter (compared to a context bound) when we actually use the the type class instance?